### ISSUE TITLE
attachments and images not loading after update to 1.8.103

### ISSUE THREAD

**Issue by Io-Qo** [freescout-helpdesk/freescout#3453]
Hey,
after updating yesterday (using git pull) to the latest version every request for attachments or image files results in an http 500 error.
PHP version: PHP 8.2.11
FreeScout version: 1.8.103

What I see in /var/log/apache/access.log
` [12/Oct/2023:10:15:01 +0000] "GET /storage/mailboxes/871b1aec.png HTTP/1.1" 500 5149 "https://mail.x.x/mailbox/10/162" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"`

The actual files do exists, permissions seem to be fine as well, I ran "chown -R www-data:www-data /var/www/html" as well to debug.
The laravel.log is clean, so is the apache error.log. I use cloudflare but have purged cache and disabled it completely, didn't help. 

I have no idea whats wrong, any tips how to debug this?

**Comment by Io-Qo** [comment]
Awesome, I wasn't aware headers had to be enabled, that fixed it. Thanks for the rapid help!

**Comment by nooblag** [comment]
Hi there, I have a similar problem, which may or may not be connected to the update (because unsure if it was happening before), I cannot upload images to a conversation, the HTTP response is "413 Request Entity Too Large" instead of 500.

I've updated the nginx conf as [here](https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#61-nginx), and restarted, but issue still appears.

System is:
- Ubuntu 22.04.3 LTS
- Nginx: 1.18.0
- PHP 8.2.10
- MariaDB 15.1

I have large settings in php.ini for use with Nextcloud on the same system:
```
max_execution_time = 3600
max_input_time = 3600
post_max_size = 16G
upload_max_filesize = 16G
max_file_uploads = 512
```

Any ideas?

I also have the `Block External Images` module installed, if that's a consideration, but temporarily disabling it did not make any difference.

**Comment by freescout-helpdesk** [comment]
It's not related to the current issue. See https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#-error-when-uploading-attachments-client-intended-to-send-too-large-body

**Comment by nooblag** [comment]
Ah, yes. Thank you. `client_max_body_size 20M;` needed to be added to the Freescout nginx conf. Perhaps you could consider adding that to the default config in your [docs](https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#61-nginx)?

Thank you!

**Comment by freescout-helpdesk** [comment]
Yes, added.

**Comment by Io-Qo** [comment]
I still get this error in the browser console when trying to load external images:

`Refused to run the JavaScript URL because it violates the following Content Security Policy directive: "script-src 'self' 'nonce-xxx'". Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present.`

How can I fix that? I'm still on 1.8.103

**Comment by freescout-helpdesk** [comment]
The error is not related to external images. Create a new issue.
